templates/content/templates/celex_classes/main.html

```diff
@@ -22,15 +21,15 @@ <h1>CELEX Classes</h1>
     ?parent skos:member ?collection
   }
 } ORDER by ?order" %}
-{% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+{% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query, **conf.celexes).fetch_tabular() %}
```

Comment:
I wonder why do we need to unpack the dict there, but I guess I did not yet reach to see the macro implementation yet.